docs/dev/compiler-toolchain/llvm.md

```diff
@@ -0,0 +1,22 @@
+# LLVM Framework
+
+[The LLVM framework](https://github.com/matter-labs/era-compiler-llvm) is an excellent choice for developing toolchains for smart contract languages. Its powerful intermediate representation (IR) allows developers to design, implement, and optimize efficient
+language-specific features while benefiting from the extensive LLVM ecosystem. This ensures high-performance execution,
+improved portability, and seamless integration with existing LLVM-based tools. Furthermore, the modularity and
+extensibility of LLVM make it easier to support new smart contract languages with LLVM front-ends.
+
+Additionally, LLVM improves on efficiency when compared to the original EVM pipeline, as we can take advantage of the
```

Comment:
“on” in “improves on efficiency” is unidiomatic, and can be removed. “when” can be deleted without loss of meaning, and tightens the style.

Reply:
Indeed 'improves on' is less common. I agree with Flash; let's keep the language as simple as possible.

Reply:
9b9b5715